---
layout: page
title: About me
subtitle: Bioinformatics | Parasitology | Molecular Biology
cover-img: /assets/img/tryps.png
---

I'm a postdoctoral researcher in the Matthews lab, in Edinburgh. I use bioinformatics and molecular techniques to understand the evolution and developmental biology of trypanosomes, mainly _Trypanosoma brucei_. My work can be split into three projects.

__Mechanisms of life cycle simplification__

African trypanosomes undergo development to transmissible stumpy forms in their mammalian host to favour uptake by their tsetse fly vector. However, subspecies of _T. brucei_ have simplified their lifecycle by escaping dependence on tsetse allowing an expanded geographical range to South America, Asia and Europe, with direct transmission achieved via biting flies or through sexual transmission between animals. Concomitantly, stumpy formation is lost, and the isolates are described as monomorphic. Through genomic analysis of distinct field isolates I identified and functionally confirmed molecular changes that accompany the loss of the stumpy transmission stage. Further, by laboratory selection, I identified reversible steps in the initial development of monomorphism. This identifies a trajectory of events that simplify the trypanosome life cycle with an impact on disease spread, vector control strategies, geographical range and virulence.

Watch out for a publication on this project. The code can be found [here](https://github.com/goldrieve/Mechanisms-of-life-cycle-simplification).

_Oldrieve G, et al. (In prep). Mechanisms of life-cycle simplification in field-derived and laboratory-selected trypanosomes._

_Oldrieve G, Verney M, Jaron KS, Hebert L, Matthews KR. 2021. Monomorphic Trypanozoon: towards reconciling phylogeny and pathologies. doi/10.199/mgen.0.000632._

__COBALT__ 

_Trypanosoma brucei_ avoid host immunity by periodically changing the proteins they express on their surface – a phenomenon called antigenic variation (AV). _T. brucei_ cause human disease, although their greatest impact is through livestock disease which significantly limits economic prosperity in African countries. The molecular mechanisms of antigen switching in trypanosomes have been extensively studied over the past three decades providing a classic model for AV. However, several key components of this model have been challenged through recent discoveries. This includes a new appreciation of the importance of gene mosaics in generating new variants, as well as the contributions of parasite development and body compartmentation to the infection dynamic. It has become clear that the existing infection model in mice poorly reflects the dominant infections found in livestock.

I have developed a reproducible pipeline to analyse amplicon sequencing of AV genes in _T. brucei_. The pipeline is freely available [here](https://github.com/goldrieve/vsgseq2) and you can follow a tutorial [here](https://github.com/goldrieve/vsgseq2). Watch out for more updates soon!

__The genomic basis of host and vector specificity in non-pathogenic trypanosomatids__

Trypanosoma theileri, a non-pathogenic parasite of bovines, has a predicted surface protein architecture that likely aids survival in its mammalian host. Their surface proteins are encoded by genes which account for ∼10% of their genome. A non-pathogenic parasite of sheep, Trypanosoma melophagium, is transmitted by the sheep ked and is closely related to T. theileri. To explore host and vector specificity between these species, we sequenced the T. melophagium genome and transcriptome and an annotated draft genome was assembled. T. melophagium was compared to 43 kinetoplastid genomes, including T. theileri. T. melophagium and T. theileri have an AT biased genome, the greatest bias of publicly available trypanosomatids. This trend may result from selection acting to decrease the genomic nucleotide cost. The T. melophagium genome is 6.3Mb smaller than T. theileri and large families of proteins, characteristic of the predicted surface of T. theileri, were found to be absent or greatly reduced in T. melophagium. Instead, T. melophagium has modestly expanded protein families associated with the avoidance of complement-mediated lysis. We propose that the contrasting genomic features of these species are linked to their mode of transmission from their insect vector to their mammalian host.

_Oldrieve G, Malacart B, López-Vidal J, Matthews KR. 2022. The genomic basis of host and vector specificity in non-pathogenic trypanosomatids. doi.org/10.1242/bio.059237._
