\documentclass[a4paper,11pt]{article}

%-----------------------------------------------------------
% FONT
%-----------------------------------------------------------
\usepackage[default]{helvet} 
\renewcommand{\familydefault}{\sfdefault}

%-----------------------------------------------------------
% PACKAGES
%-----------------------------------------------------------
\usepackage{url}
\usepackage{parskip}
\usepackage{graphicx}
\usepackage[usenames,dvipsnames]{xcolor}
\usepackage[scale=0.9]{geometry}
\usepackage{tabularx}
\usepackage{enumitem}
\usepackage{titlesec}
\usepackage{multicol}
\usepackage{fontawesome5}
\usepackage[unicode]{hyperref}
\usepackage[most]{tcolorbox}

% link styling
\definecolor{linkcolour}{rgb}{0.0,0.2,0.0}
\hypersetup{colorlinks, urlcolor=linkcolour, linkcolor=linkcolour}

% job environments
\newenvironment{joblong}[2]{
  \begin{tabularx}{\linewidth}{@{}l X r@{}}
  \textbf{#1} & & #2 \\[2pt]
  \end{tabularx}
  \begin{itemize}[leftmargin=1.2em, itemsep=3pt, label=-]
}{
  \end{itemize}
}

\newenvironment{jobshort}[2]{
  \begin{tabularx}{\linewidth}{@{}l X r@{}}
  \textbf{#1} & & #2 \\[2pt]
  \end{tabularx}
}{}

% section style
\titleformat{\section}{\large\bfseries\uppercase}{}{0em}{}[\titlerule]
\titlespacing{\section}{0pt}{8pt}{6pt}

% Define a skill box style
\newtcbox{\skillbox}[1][]{%
  on line,
  colframe=black!40,
  colback=black!5,
  boxrule=0.4pt,
  arc=3pt,
  boxsep=2pt,
  left=4pt,
  right=4pt,
  top=2pt,
  bottom=2pt,
  #1}

%-----------------------------------------------------------
\begin{document}
\pagestyle{empty}

%-----------------------------------------------------------
% HEADER
%-----------------------------------------------------------

\begin{center}
    {\Huge \textbf{Guy Oldrieve, PhD}} \\[6pt]
    \href{mailto:oldrieve2@hotmail.com}{\faEnvelope\ oldrieve2@hotmail.com}  \ $|$ \ 
    \href{https://github.com/goldrieve}{\faGithub\ goldrieve}  \ $|$ \ 
    \href{https://linkedin.com/in/guy-oldrieve}{\faLinkedin\ guy-oldrieve}
    \end{center}

%-----------------------------------------------------------
\begin{multicols}{2}
\subsection*{Summary}
\item Bioinformatician (6+ years) specialising in data-driven approaches to disease research and discovery. Skilled in developing, optimising, and validating scalable multi-omic pipelines. Currently focused on implementing nf-testâ€“based CI for ISO-certified Nextflow pipelines, ensuring reproducibility, reliability, and compliance.

\columnbreak

\subsection*{}
\item \skillbox{Python} \skillbox{R} \skillbox{bash} \skillbox{Nextflow} \skillbox{nf-test} \skillbox{GitHub} \skillbox{AWS/HPC} \skillbox{GH Actions}
\end{multicols}

%-----------------------------------------------------------
\section{Experience}

\begin{joblong}{Bioinformatician | Animal and Plant Health Agency}{2025-Present}
\item Reviewed and streamlined in-house pipelines for sequencing QC and phylogenetic analysis of bovine TB isolates
  \begin{itemize}[leftmargin=2em, itemsep=2pt, label=$\circ$]
    \item Developed and improved nf-test CI for ISO-17025 certified Nextflow pipelines
    \item Created btb-sub, a Nextflow pipeline which automates the submission of sequencing data to ENA
  \end{itemize}
\end{joblong}

\begin{joblong}{Postdoctoral Research Associate | The University of Edinburgh}{2023-2025}
\item Created vsgseq2, a Nextflow pipeline for analysing population-level antigenic variation in T. brucei\textsuperscript{1}
    \begin{itemize}[leftmargin=2em, itemsep=2pt, label=$\circ$]
        \item Integrated nf-test unit/integration testing and Docker for reproducible distribution
        \item Applied vsgseq2 to hundreds of chronic bovine infection samples
        \item Generated high-quality genome assemblies from PacBio, ONT, and Hi-C data to understand mechanisms of antigenic variation
    \end{itemize}
\item Developed a MinION-based diagnostic for real-time outbreak detection of T. brucei
    \begin{itemize}[leftmargin=2em, itemsep=2pt, label=$\circ$]
        \item Built a Python based machine learning app for phenotype prediction and outbreak risk
        \item Performed amplicon generation and ONT sequencing
        \item Supervised and taught two students in both wet lab and bioinformatics
    \end{itemize}
\item Delivered HPC training and managed group environments
\item Co-authored the first profile of the T. brucei cell cycle with single-cell RNAseq\textsuperscript{4}
\item Serve on the British Society for Parasitology council, promoting adoption of new technologies
\item Reviewer: Trends in Parasitology, PLOS NTD, Frontiers in Cellular and Infection Microbiology and Parasitology Research
\end{joblong}

\begin{joblong}{PhD in Hosts, Pathogens, and Global Health | The University of Edinburgh}{2018-2023}
\item Built a Snakemake pipeline to investigate life cycle simplification in T. brucei\textsuperscript{3,6}
\begin{itemize}[leftmargin=2em, itemsep=2pt, label=$\circ$]
  \item Identified mutations disrupting quorum sensing via analysis of genomic and transcriptomic data 
  \item Validated predictions through CRISPR/Cas9 genetic manipulation
\end{itemize}
\item Sequenced and assembled a parasite genome remotely using MinION during pandemic restrictions\textsuperscript{5}
\item Presented research at international conferences to scientific and general audiences
\item Supervised a student in genome assembly and comparative genomics, resulting in a co-authored publication
\end{joblong}

\begin{joblong}{MSc in Hosts, Pathogens and Global Health | The University of Edinburgh}{2018-2019}
\item Produced high-quality bacterial genome assemblies to investigate isolate relationships\textsuperscript{2}
\item Developed an R-based data mining pipeline to identify severe malaria biomarkers from unstructured datasets
\end{joblong}

\begin{joblong}{Research Assistant | Cardiff University}{2013-2018}
\item Applied Python-based machine learning to filter contaminated transcriptomic data
\end{joblong}

%-----------------------------------------------------------
\section{Education}
\begin{itemize}[leftmargin=1.2em, itemsep=3pt, label=-]
\item \begin{tabularx}{\linewidth}{@{}X r@{}}
PhD in Hosts, Pathogens, and Global Health at University of Edinburgh \textbf{(Ker Memorial Prize)} & 2018-2023 \\
\end{tabularx}
\item \begin{tabularx}{\linewidth}{@{}X r@{}}
MSc in Hosts, Pathogens and Global Health at University of Edinburgh \textbf{(Merit)} & 2018-2019 \\
\end{tabularx}
\item \begin{tabularx}{\linewidth}{@{}X r@{}}
Integrated Master's in Biology (MBiol) at Cardiff University \textbf{(First-class)} & 2013-2018 \\
\end{tabularx}
\end{itemize}

%-----------------------------------------------------------
\section{Publications}
\begin{enumerate}[leftmargin=1.2em, itemsep=3pt]
\item \textbf{Oldrieve G.}, et al. (2025). vsgseq2: an updated pipeline for analysis of the diversity and abundance of population-wide Trypanosoma brucei VSG expression. In Prep.: Wellcome Open Research
\item Vancaester E., \textbf{Oldrieve G.}, et al. (2025) Ghosts of symbionts past: The hidden history of the dynamic association between filarial nematodes and their Wolbachia endosymbionts. Accepted: G3
\item \textbf{Oldrieve G.}, et al. (2024) Mechanisms of life cycle simplification in African trypanosomes. Nature Communications
\item Briggs E., Marques C., \textbf{Oldrieve G.}, et al. (2023). Profiling the Trypanosoma brucei cell cycle using single-cell transcriptomics. eLife
\item \textbf{Oldrieve G.}, et al. (2022). The genomic basis of host and vector specificity in non-pathogenic trypanosomatids. Biology Open
\item \textbf{Oldrieve G.}, et al. (2021). Monomorphic Trypanozoon: towards reconciling phylogeny and pathologies. Microbial Genomics
\end{enumerate}

%-----------------------------------------------------------
\section{Awards}
\begin{itemize}[leftmargin=1.2em, itemsep=3pt, label=-]
\item \begin{tabularx}{\linewidth}{@{}X r@{}}
\textbf{Ker Memorial Prize}: Best PhD thesis in infectious diseases, University of Edinburgh & (2024) \\
\end{tabularx}
\item \begin{tabularx}{\linewidth}{@{}X r@{}}
\textbf{Best poster}: Edinburgh Infectious Diseases symposium & (2020) \\
\end{tabularx}
\item \begin{tabularx}{\linewidth}{@{}X r@{}}
\textbf{Best performance}: Integrated master's in biology & (2018) \\
\end{tabularx}
\end{itemize}

%-----------------------------------------------------------
\section{Interests}
\begin{itemize}[leftmargin=1.2em, itemsep=3pt, label=-]
\item \begin{tabularx}{\linewidth}{@{}X r@{}}
Endurance Events: Perranporth Triathlon and Ljubljana, Loch Ness and Edinburgh marathon & (2023-Present) \\
\end{tabularx}
\item \begin{tabularx}{\linewidth}{@{}X r@{}}
Captain, Lismore RFC: Led a squad of 50+ players & (2020-2023) \\
\end{tabularx}
\end{itemize}

\end{document}